client/src/pages/Restaurant/FoodList/FoodCard.js

```diff
@@ -1,5 +1,57 @@
-function FoodCard() {
-  return <div>FoodCard</div>;
+function FoodCard({ name, price, description, imageUrl }) {
+  const classes = {
+    cardContainer: {
+      boxSizing: "border-box",
+      width: "100%",
+      height: "68px",
+      border: "2px solid #5a5a5a",
+      padding: "2px",
+      display: "flex",
+      gap: "0.25rem",
+      backgroundColor: "hsl(35,0%,97%)",
+    },
+    foodInfo: {
+      maxWidth: "740px",
+    },
+    foodName: {
+      fontSize: "1rem",
+      fontWeight: "bold",
+      overflow: "hidden",
+      whitespace: "nowrap",
+      textOverflow: "ellipsis",
+      height: "30%",
+    },
+    foodPrice: {
+      fontSize: "1rem",
+      marginLeft: "auto",
+      padding: "15px",
+    },
+    foodDescription: {
+      fontSize: "0.8rem",
+      overflow: "auto",
+      whiteSpace: "normal",
+      textOverflow: "ellipsis",
+      height: "45px",
```

Comment:
Make this like .75 rem with color gray

What's the height for?

Reply:
Similar reasoning to the foodName height but the height here is for making sure the food description doesn't spill over outside the food card. Instead, it now provides a little scrollbar for long descriptions.